Protection Profile/BiocPP.adoc

```diff
@@ -587,63 +576,61 @@ The objectives for the biometric system and its operational environment are cons
-** Plaintext biometric data shall never be accessible from outside the Separate Execution Environment, and any entities outside the Separate Execution Environment can only access the result of process of biometric data (e.g. success or failure of biometric verification) through the interface provided by the Biometric System.
+However, the Biometric System shall use this SEE correctly to protect biometric data and satisfy the following requirements:
 
-* The Biometric System shall not transmit any plaintext biometric data outside of the Separate Execution Environment.
+* The Biometric System shall process any plaintext biometric data (e.g. capturing biometric characteristic, creating samples, features and templates) for biometric enrolment and verification within the boundary of the SEE. This implies that:
+** Any part of the Biometric System that processes plaintext biometric data shall be within the boundary of the SEE. For example, the biometric capture sensor shall be configured to be within the boundary of the SEE, so that only the SEE can access to the sensor and the data captured. Any software modules that process plaintext biometric data shall run within the boundary of the SEE.
+** Plaintext biometric data shall never be accessible from outside the SEE, and any entities outside the SEE can only access the result of process of biometric data (e.g. success or failure of biometric verification) through the interface provided by the Biometric System.
 
-If the Biometric System stores any part of the biometric data outside the Separate Execution Environment, the Biometric System shall protect such data so that any entities running outside the Separate Execution Environment can not get access to any plaintext biometric data. ST author shall explain what biometric data resides outside the Separate Execution Environment as required by <<BIOSD>> and if no data resides outside the environment, requirements below is implicitly satisfied.
+* The Biometric System shall not transmit any plaintext biometric data outside of the SEE.
 
-* The Biometric System shall not store any plaintext biometric data outside the Separate Execution Environment. As described in this PP-Module Section TOE design, the Biometric System can store templates in the enrolment database. The Biometric System shall encrypt templates using cryptographic service provided by the mobile device within the Separate Execution Environment before storing them in the database, even if the mobile device storage itself is encrypted by the mobile device.
+If the Biometric System stores any part of the biometric data outside the SEE, the Biometric System shall protect such data so that any entities running outside the SEE can not get access to any plaintext biometric data. ST author shall explain what biometric data resides outside the SEE as required by <<BIOSD>> and if no data resides outside the environment, requirements below is implicitly satisfied.
```

Comment:
```suggestion
If the Biometric System stores any part of the biometric data outside the SEE, the Biometric System shall protect such data so that any entities running outside the SEE cannot get access to any plaintext biometric data. ST author shall explain what biometric data resides outside the SEE as required by <<BIOSD>> and if no data resides outside the environment, the requirements below are implicitly satisfied.
```